Makefile

```diff
@@ -29,7 +29,9 @@ else
   GIT_VERSION =
 endif
 
-CFLAGS_MIN := -W -Wall -DVERSION_STRING=\"v$(VERSION)\" -DGIT_VERSION=\"$(GIT_VERSION)\"
+PIDFILE ?= /run/llmnrd/pid
```

Comment:
Let's name this `/run/llmnr.pid`, so we don't have to create leading directories and to stick with the `*.pid` file name convention other daemons seem to use.

Reply:
Using `/run/llmnr.pid` does not work when llmnr is running as non-root, as `/run` is writable by root only, that's why I added the `llmnrd` directory, which can be created writable by the `llmnrd` user in the init script.

Reply:
Good point, thanks for explaining. I wasn't considering that `/run` might not be user-writable. It's fine as is in that case.